CONTRIBUTING.mdown

```diff
@@ -9,7 +9,7 @@ To contribute on behalf of yourself, sign the individual CLA
 
 All contributions:
 
-1/ MUST be be licensed using the Apache License, Version 2.0  
-2/ authors MAY retain copyright by adding their copyright notice to the appropriate flies 
+1. Must be licensed using the Apache License, Version 2.0  
+2. Authors may retain copyright by adding their copyright notice to the appropriate flies 
 
-More information on the Apache License can be found here: http://www.apache.org/foundation/license-faq.html
+More information on the Apache License, it can be found [here](http://www.apache.org/foundation/license-faq.html)
```

Comment:
The previous wording was correct and we would prefer for this short url to show the whole link.